README.md

```diff
@@ -290,6 +290,20 @@ end
   end
   ```
 
+* `cpuaffinitiy` - Mapping of vCPUs to host CPUs. [See `vcpupin`](https://libvirt.org/formatdomain.html#elementsCPUTuning).
+
+  ```ruby
+  Vagrant.configure("2") do |config|
+    config.vm.provider :libvirt do |libvirt|
+      libvirt.cpus = 4
+      libvirt.cpuaffinitiy
```

Comment:
Typo using `cpuaffinity` here, but `cpu_affinity` for the config option below. Guessing that is the source of the failure you were seeing testing locally?